tests/compute/test_basics.py

```diff
@@ -290,6 +291,66 @@ def _check_nx_feature(nxg, nf, ef):
     assert F.allclose(g.edata['h'], F.tensor([[1., 2.], [1., 2.],
                                               [2., 3.], [2., 3.]]))
 
+def test_to_cugraph_conversion():
```

Comment:
pls move these tests into `//tests/cugraph/test_basics.py`.

Reply:
Done . Thanks for adding cugraph to the CI.